r/tests/testthat/test-dataset-write.R

```diff
@@ -703,6 +703,7 @@ test_that("Dataset min_rows_per_group", {
 
   row_group_sizes <- ds %>%
     map_batches(~ record_batch(nrows = .$num_rows)) %>%
+    (function(x) x$read_table()) %>%
```

Comment:
Would `collect()` not work here?

Reply:
Ah right I should have read your earlier comment to completion. Thanks for explaining!